forwards_paper.tex

```diff
@@ -240,6 +241,46 @@ \subsection*{Simulation benchmarks}
 
 \plr{Insert figures/tables of performance here and describe them.}
 
+The total run times for simulations with selection are shown in Figure~\ref{fig:runtimes_selection}.  When tracking
+neutral mutations (instead of the ARG), run times increase dramatically with increasing region size ($4Nr = 4Nu$ for
+these simulations).  With \fwdpy{}, simulations with $N=5 \times 10^4$ timed out for region sizes larger than $10^3$.
+With \simupop{}, we were only able to run simulations with $N=10^3$ and region sizes up to $10^4$.  ARG tracking
+dramatically improves run times for both \fwdpy{} and \simupop{} (right column of Figure~\ref{fig:runtimes_selection}).  For parameters that ran to completion when tracking neutral mutations and when tracking
+the ARG, the relative speedup due to ARG tracking is up to $\approx 50$ fold (Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, ARG tracking reduces runtimes more for larger population sizes
+(Figure~\ref{fig:relative_speedup_selection}).  We could not make such an observation with \simupop{}, which tended to
+run out of memory on our system.  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
+partially obscures an important observation that the run times appear approximately linear in region size when ARG
+tracking.  The reason for this behavior is that the simulations are generating a constant number of new edges on average
+each generation, and the number of edges is a function of $\rho$ in expectation.
```

Comment:
at first I thought that this linear-in-region-size observation should go in the run-time complexity section; but since this comes first, it's good here; we should just refer to it below.